crates/ruff_workspace/src/options.rs

```diff
@@ -2469,6 +2502,31 @@ impl PyUpgradeOptions {
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct FormatOptions {
+    /// A list of file patterns to exclude from formatting in addition to the files excluded globally (see [`exclude`](#exclude), and [`extend-exclude`](#extend-exclude)).
```

Comment:
I dislike that we have effectively three copies of this but i don't have a good solution either unfortunately